modules/backend/models/EditorSetting.php

```diff
@@ -99,6 +108,14 @@ public function initSettingsData()
         $this->html_style_paragraph = $this->makeStylesForTable($this->defaultHtmlStyleParagraph);
         $this->html_style_table = $this->makeStylesForTable($this->defaultHtmlStyleTable);
         $this->html_style_table_cell = $this->makeStylesForTable($this->defaultHtmlStyleTableCell);
+        $this->html_paragraph_formats = $this->makeFormatsForTable($this->defaultHtmlParagraphFormats);
+    }
+
+    public function afterFetch()
+    {
+        if (!isset($this->value['html_paragraph_formats'])) {
+            $this->html_paragraph_formats = $this->makeFormatsForTable($this->defaultHtmlParagraphFormats);
```

Comment:
```suggestion
            $this->html_paragraph_formats = $this->makeFormatsForTable($this->defaultHtmlParagraphFormats);
            $this->save();
```
Make sure the defaults are saved in the settings so this doesn't continue to get run on every request.